pom.xml

```diff
@@ -241,5 +241,18 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.eclipse.jetty.aggregate</groupId>
+            <artifactId>jetty-all</artifactId>
+            <version>8.1.22.v20160922</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>5.3.9</version>
+            <scope>provided</scope>
```

Comment:
So does this require to have spring-context and jetty-all within the OSGI container running? What kind of additional resource consumption (deployment size, heap) would that mean?

Reply:
Currently you already has this dependencies in runtime. You get they from activemq.

They added as "provided" for fix compilation